### PR TITLE
for #41: start metrics.md doc

### PR DESCRIPTION
**Pull request by groovecoder**
Need @harterrt and/or @ilanasegall to review. Especially the "TBD" sections.

@pdolanjski - you should check this out too.

TODOs:
* [x] `breakage` ping: `branch` and `notes` fields are missing, and `event` is empty
* [x] `comment` ping: `branch` field is missing, and `event` is empty
* [x] Should we send our own `disable` ping in addition to `user-ended-sudy`?

### CODE REVIEW

File: docs/metrics.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+```
+TBD?
+```
+
+A Redshift schema for the payload:
+
+TBD ...
+
+```lua
+local schema = {
```
Comment (harterrt) [inline docs/metrics.md]: So we're always ignoring `discarding payload.about` and `payload.study_version` (at least for this version of the data)?

File: docs/metrics.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+### Server Side
+There is currently no server side component besides existing Unified Telemtry.
+
+### Client Side
+Study will use SHIELD's Telemetry wrapper with no batching of data.  Details
+of when pings are sent are below, along with examples of the `payload` portion
+of a `shield-study` telemetry ping for each scenario.
+
+#### The user clicks "This page works well" button in the popup
```
Comment (harterrt) [inline docs/metrics.md]: When is the popup triggered?

**Review by harterrt — APPROVED** [review]
Looks good overall, Luke. A couple of questions inline. 

The ETL to get this table into re:dash should be pretty straightforward and very similar to the containers code. When will this experiment go live?

File: docs/metrics.md (hunk 1 above)
Comment (groovecoder) [reply docs/metrics.md]: I'm not planning to use them in the breakage dashboard queries or reports, but I assume they are useful for other (ATMO?) querying of shield study data?

File: docs/metrics.md (hunk 2 above)
Comment (groovecoder) [reply docs/metrics.md]: I added more details about when the pop-up is triggered.

**Comment by groovecoder** [comment]
Updated to final metrics.md and matching Telemetry ping code.

Want to launch this study ASAP as it uses SDK code, which we lose in August.

**Comment by groovecoder** [comment]
There may be an even more similar ETL job; where is the ETL job that was used for Blok/Tracking Protection Test Pilot experiment? The job that populated the `blok_daily` table.

**Comment by harterrt** [comment]
@groovecoder - do you know who wrote the `block_daily` job or where that code lives? I'm unfamiliar with the data.

**Comment by groovecoder** [comment]
I'm pretty sure it was rrayborn?

**Pull request merged**